src/helpers/augmentedReality/supportedDevice.js

```diff
@@ -0,0 +1,11 @@
+import { osVersion } from 'expo-device';
+import { useContext } from 'react';
+
+import { device } from '../../config';
+import { SettingsContext } from '../../SettingsProvider';
+
+export const supportedDevice = () => {
+  const { globalSettings } = useContext(SettingsContext);
+
+  return { isSupported: osVersion >= globalSettings.augmentedReality[device.platform] };
```

Comment:
I think this should do the trick.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/593/commits/5b917e75ba973aa000fa55701615b059d8114296

Reply:
ok but i suggest to add a `?` as well for `augmentedReality[device.platform]` to ensure that nothing breaks if `device?.platform` returns something that is not included in `augmentedReality`.

and the first question? where does `augmentedReality` come from in the global settings?

Reply:
It comes from the augmentedReality object in globalSettings in int/Development server.

Reply:
I'm not happy with this code. I think it could have been better.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/593/commits/735aa35dda1b5631991b68760609d245dd566f5e

Reply:
i think the changes are very clear and straight forward now at the end 👍